contracts/masset/platform-integrations/CompoundIntegration.sol

```diff
@@ -239,6 +245,50 @@ contract CompoundIntegration is InitializableAbstractIntegration {
         uint256 exchangeRate = _cToken.exchangeRateStored();
         // e.g. 1e18*1e18 / 205316390724364402565641705 = 50e8
         // e.g. 1e8*1e18 / 205316390724364402565641705 = 0.45 or 0
-        amount = _underlying.mul(1e18).div(exchangeRate);
+       amount = _underlying.mul(1e18).div(exchangeRate);
+    }
+
+    /**
+     * @dev Checks whether a claim should be made
+     * This compares the block.timestamp with a somewhat random time
+     * Adds randomness by muliplying the 1 hour delay between 1x and 3x
+     * @param _bAsset Address for the bAsset
+     */
+    function _checkClaim(address _bAsset)
+        internal
+    {
+        uint256 salt = uint256(keccak256(abi.encodePacked(blockhash(block.number)))).mod(3000000);
+        uint256 timeDelay = ((uint256(1 hours)).mul(salt)).div(1000000);
```

Comment:
What number is this supposed to produce?

Reply:
1 hour with a multiplier between 1 and 3